ci/Jenkinsfile

```diff
@@ -1,2 +1,2 @@
-golang([test: "go test -v -timeout 20m ./..."])
+golang([test: "go test -v -loglevel ERROR ./..."])
```

Comment:
why have a different loglevel between jenkins and travis?

Reply:
temporary